doc/source/whatsnew/v1.2.0.rst

```diff
@@ -24,27 +24,45 @@ prevent accidental introduction of duplicate labels, which can affect downstream
 
 By default, duplicates continue to be allowed.
 
-.. ipython:: python
+.. code-block:: ipython
 
-   pd.Series([1, 2], index=['a', 'a'])
+    In [1]: pd.Series([1, 2], index=['a', 'a'])
```

Comment:
technically you don't need to change the block on data creation, but nbd

Reply:
By keeping it in a single directive, it gives a consistent code block (not 2 different code blocks with different look or inconsistent number), so in this case nicer I think.